docs/editors/vscode.md

```diff
@@ -120,29 +124,36 @@ without opening the sidebar.
 As you type, the symbol outline is also visible at the top of the file.
 ![Document Symbols Outline](https://i.imgur.com/L217n4q.png)
 
-## Enable on type formatting for multiline string formatting
+```scala mdoc:parent-lenses:vscode
 
-![pipes](https://i.imgur.com/iXGYOf0.gif)
+```
 
-To properly support adding `|` in multiline strings we are using the
-`onTypeFormatting` method. To enable the functionality you need to enable
-`onTypeFormatting` inside Visual Studio Code.
+```scala mdoc:new-project:vscode
+
+```
 
-This needs to be done in settings by checking `Editor: Format On Type`:
+## On type formatting for multiline string formatting
+
+![on-type](https://imgur.com/a0O2vCs.gif)
+
+To properly support adding `|` in multiline strings we are using the
+`onTypeFormatting` method. The functionality is enabled by default, but you can
+disable/enable `onTypeFormatting` inside Visual Studio Code settings by checking
+`Editor: Format On Type`:
 
-![on-type](https://i.imgur.com/4eVvSP5.gif)
+![on-type-setting](https://i.imgur.com/s6nT9rC.png)
 
-## Enable formatting on paste for multiline strings
+## Formatting on paste for multiline strings
 
 Whenever text is paste into a multiline string with `|` it will be properly
 formatted by Metals:
 
-![format-on-paste](https://i.imgur.com/yJLAIxQ.gif)
+![format-on-paste](https://i.imgur.com/fF0XWYC.gif)
```

Comment:
Added better gifs, because the previous ones were bother me quite a bit :sweat_smile: